### ISSUE TITLE
PytestUnknownMarkWarning with pytest 4.3.0

### ISSUE THREAD

**Issue by languitar** [omarkohl/pytest-datafiles#8]
With pytest 4.3.0 I am constantly seeing the following warning when executing tests using datafiles:
```
.tox/test-py37/lib/python3.7/site-packages/_pytest/mark/structures.py:324
  xxxx/.tox/test-py37/lib/python3.7/site-packages/_pytest/mark/structures.py:324: PytestUnknownMarkWarning: Unknown pytest.mark.datafiles - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,
```
The plugin works as expected though

**Comment by darrenleeweber** [comment]
Same here:

```
/home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/_pytest/mark/structures.py:324
/home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/_pytest/mark/structures.py:324
/home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/_pytest/mark/structures.py:324
/home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/_pytest/mark/structures.py:324
  /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/_pytest/mark/structures.py:324: PytestUnknownMarkWarning: Unknown pytest.mark.datafiles - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

```

Versions:
```

$ pytest --version
This is pytest version 4.5.0, imported from /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest.py
setuptools registered plugins:
  pytest-forked-1.0.2 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_forked/__init__.py
  pytest-benchmark-3.2.2 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_benchmark/plugin.py
  pytest-voluptuous-1.1.0 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_voluptuous/plugin.py
  pytest-cov-2.7.1 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_cov/plugin.py
  pytest-datafiles-2.0 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_datafiles.py
  pytest-mock-1.10.4 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_mock.py
  pytest-datadir-1.3.0 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pytest_datadir/plugin.py
  pytest-xdist-1.28.0 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/xdist/plugin.py
  pytest-xdist-1.28.0 at /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/xdist/looponfail.py

$ python --version
Python 3.6.7

$ pip --version
pip 19.1.1 from /home/joe/.cache/pypoetry/virtualenvs/org-obs-py3.6/lib/python3.6/site-packages/pip (python 3.6)
```

**Comment by omarkohl** [comment]
According to the changelog this is a new feature of [pytest 4.5.0](https://docs.pytest.org/en/latest/changelog.html#pytest-4-5-0-2019-05-11) to avoid errors due to typos in markers. Are you sure you are using 4.3.0 .

**Comment by languitar** [comment]
Sorry, might be that I picked up the wrong pytest version from a different tox environment.

**Comment by drorata** [comment]
I have this issue when running:

```
platform darwin -- Python 3.7.4, pytest-5.0.1, py-1.8.0, pluggy-0.13.0
rootdir: /Users/user/my-location, inifile: pytest.ini
plugins: datafiles-2.0, dotenv-0.4.0, cov-2.7.1
```

**Comment by drorata** [comment]
I could "fix" it by adding:

```
markers =
    datafiles: load datafiles
```

to my `pytest.ini`.

**Comment by hf-krechan** [comment]
Just in case if someone is using `pyproject.toml`:
```toml
[tool.pytest.ini_options]
markers = ["datafiles: load datafiles"]
```

**Comment by omarkohl** [comment]
Should be fixed in commit f95315ee2dff86ffeb17a41c203d54c1fc1405f0 . I will make a release soon.

**Comment by omarkohl** [comment]
Released in 2.0.1
